File: Arrays/Two_Number_Sum/Java/TwoNumSumBruteForce.java
import java.util.*;

class twoNumSumBruteForce {
  public static void main(String[] args) {
    int [] numbers = new int[] {1, 2, 4, 5, -5, -2};
    int targetSum = 2;

    twoNumSum(numbers, targetSum);
  };
  
  public static int[] twoNumSum(int[] numbers, int targetSum) {
    for (int i = 0; i < numbers.length - 1; i++) {
      int firstNum = numbers[i];
      for (int j = i + 1; j < numbers.length; j++) {
        int secondNum = numbers[j];
        
        if (firstNum + secondNum == targetSum) {
          int[] result = new int[] {firstNum, secondNum};
          System.out.println(Arrays.toString(result));
          System.out.println("You got the answer!");
          return new int[] {firstNum, secondNum};
        }
      }
    }
    // System.out.println(new int[0]);
    return new int[0];
  };
};

File: Arrays/Two_Number_Sum/JavaScript/twoNumSumBruteForce.js
function twoNumSumBruteForce(numbers, targetSum) {
  for (let i = 0; i < numbers.length - 1; i++) {
    const firstNum = numbers[i];
    
    for (let j = i + 1; j < numbers.length; j++) {
      const secondNum = numbers[j];

      if (firstNum + secondNum == targetSum) {
        console.log([firstNum, secondNum])
        console.log("You got it!")
        return [firstNum, secondNum];
      };
    };
  };
  return [];
}

let numbers = [1, 2, 4, 5, -5, -2];
let targetSum = 2;

twoNumSumBruteForce(numbers, targetSum);


exports.twoNumSumBruteForce = twoNumSumBruteForce;
